### PR TITLE
[test] Ensure that the first token generation is not included into TPOT

### PR DESCRIPTION
**Pull request by pavel-esir**
CVS-155098

### CODE REVIEW

File: src/cpp/src/perf_metrics.cpp [hunk 1]
```diff
@@ -101,7 +101,7 @@ void PerfMetrics::evaluate_statistics(std::optional<TimePoint> start_time) {
 
         auto ttft = tok_times[0] - start_time_val;
         raw_metrics.m_times_to_first_token = std::vector<MicroSeconds>();
-        raw_metrics.m_times_to_first_token.emplace_back(ttft / batch_sizes[0]);
+        raw_metrics.m_times_to_first_token.emplace_back(ttft);
```
Comment (pavel-esir) [inline src/cpp/src/perf_metrics.cpp:104]: If we have batch 10 and it takes 1 sec until the first token is generates then time to the first token is still 1 sec not 100 ms! Therefore i removed `/ batch_sizes[0]`

Comment (Wovchena) [reply src/cpp/src/perf_metrics.cpp:104]: I agree, but should we notify llm_bench about the change?

Comment (pavel-esir) [reply src/cpp/src/perf_metrics.cpp:104]: I asked @eaidova what she thinks. Let's wait her answer

Comment (eaidova) [reply src/cpp/src/perf_metrics.cpp:104]: @peterchen-intel  @wgzintel  do you have objections? What the reason to divide first token latency on batch?

**Review by Wovchena — APPROVED** [review]

**Review by eaidova — APPROVED** [review]

File: src/cpp/src/perf_metrics.cpp (hunk 1 above)
Comment (pavel-esir) [reply src/cpp/src/perf_metrics.cpp:104]: As i see from blame, it was added by @ialbrecht.

**Pull request merged**